ClosedXML/Excel/Cells/IXLCell.cs

```diff
@@ -383,6 +383,10 @@ public interface IXLCell
 
         XLTableCellType TableCellType();
 
+        string ToString();
```

Comment:
Is it necessary to define a `ToString()` method in the interface? All CLR types have it anyway.

Reply:
True!